src/Config.php

```diff
@@ -205,11 +206,11 @@ public function unset(string $key): bool
      * Load configuration options from a file or directory.
      *
      * @param string $path Path to configuration file or directory
-     * @param string $prefix A key under which the loaded config will be nested
-     * @param bool $override Whether or not to override existing options with
+     * @param string|null $prefix A key under which the loaded config will be nested
```

Comment:
This should never be `null` if defined.

```suggestion
     * @param string $prefix A key under which the loaded config will be nested
```